app/components/avo/panel_component.html.erb

```diff
@@ -20,9 +20,9 @@
     <% end %>
   <% end %>
   <% if body? %>
-    <div data-target="panel-body" class="flex flex-col sm:flex-row space-y-4 sm:space-y-0 sm:gap-4 w-full <% if sidebar? %> has-sidebar <% end %>">
-      <div class="relative flex-1 w-full <% if sidebar? %> sm:w-2/3 <% end %>">
-        <% # The body is wrapped inside another div in order to avoid long & tall panels next to sidebars when the sidebar taller. %>
+    <div data-target="panel-body" class="<%= class_names("flex flex-col sm:flex-row space-y-4 sm:space-y-0 sm:gap-4 w-full", "has-sidebar": sidebar?) %>">
+      <div class="<%= class_names("relative flex-1 w-full", "sm:w-2/3": sidebar?) %>">
+        <%# The body is wrapped inside another div in order to avoid long & tall panels next to sidebars when the sidebar taller. %>
```

Comment:
here we had a prasing error because of the extra space `<% #` rather than  `<%`

while on this partial I updated the `class` attribute to use `class_names` helper rather than an `<% if %> .... <% end %>` clause inside a string